pkg/v13/kubeconfig/current.go

```diff
@@ -0,0 +1,27 @@
+package kubeconfig
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/giantswarm/microerror"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/errors"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+)
+
+func (r *StateGetter) GetCurrentState(ctx context.Context, obj interface{}) ([]*corev1.Secret, error) {
+	r.logger.LogCtx(ctx, "level", "debug", "message", fmt.Sprintf("finding kubeconfig secret %#q", r.resourceName))
```

Comment:
What will be the value of `r.resourceName`? We need to create a kubeconfig secret per tenant cluster. So I think we should use `[cluster ID]-kubeconfig` as the secret name. WDYT?

Reply:
That's better, I'll change that.